app/modules/annotations/models.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""
+Annotations database models
+--------------------
+"""
+
+from sqlalchemy_utils import Timestamp
+
+from app.extensions import db
+
+import uuid
+
+
+class Annotation(db.Model, Timestamp):
+    """
+    Annotations database model.
+    """
+
+    guid = db.Column(
+        db.GUID, default=uuid.uuid4, primary_key=True
+    )  # pylint: disable=invalid-name
+
```

Comment:
may i propose a BigInt `version` as well?  i have been trying to maintain this value between EDM and their houston-based feather equivalents, for some vague (future-ish) way of testing for stale or otherwise out of sync cousin objects.  since it looks like i am going to have a shadow of Annotation in EDM as well, i would be keen to have it on here too.

Reply:
Yup, no problem, that guid was feeling very lonely.